### PR TITLE
Docs: update "from scratch" guide to fix my own comments

### PR DESCRIPTION
**Pull request by thockin**
Still more to do, but this fixes some of the more obvious issues around net.

### CODE REVIEW

File: docs/getting-started-guides/scratch.md [hunk 1]
```diff
@@ -342,6 +354,7 @@ so that kube-proxy can manage iptables instead of docker.
 
 You may want to increase the number of open files for docker:
    - `DOCKER_NOFILE=1000000`
+Where this config goes depends on your node OS.  For example, GCE's Debian-based distro uses `/etc/default/docker`.
```
Comment (mikedanese) [inline docs/getting-started-guides/scratch.md]: Do you want an extra newline before this line? This is formatted on the same line as - `DOCKER_NOFILE=...` but applies to the entire docker config section.

**Comment by erictune** [comment]
LGTM. thanks for fixing.

File: docs/getting-started-guides/scratch.md (hunk 1 above)
Comment (thockin) [inline docs/getting-started-guides/scratch.md]: fixed

**Comment by thockin** [comment]
Docs merge - who wants to OK it?

**Comment by mikedanese** [comment]
LGTM

**Pull request merged**